### DISCUSSION TITLE
aws 란 무엇인가

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#32] [Q&A] [answered]
aws 란 무엇인가

**Comment by tjdud22** [comment, marked as answer]
aws는 Amazon Web Services의 약자로, 아마존닷컴에서 운영하는 Cloud Computing Platform이다.
다른 웹 사이트나 클라이언트측 응용 프로그램에 대해 온라인 서비스를 제공하고 있다.
